packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -590,30 +587,65 @@ export class AtomicSearchBox {
     if (index === lastIndex && item.hideIfLast) {
       return null;
     }
+
+    const isButton = item.onSelect || item.query;
+    const commonAttributes = {
```

Comment:
There's a bit of an overload of ternary operation in there 😂 .

I think it would be more readable if the code was more verbose, with each section extracted into small private function:

```
getClass();
setReference();
setAriaLabel();
```

And just more good old boring `if (x) {return y}`